web/src/helpers/fetcher.ts

```diff
@@ -32,3 +32,7 @@ export const fetcher = async (input: RequestInfo, init: RequestInit) => {
     throw new FetcherError([(error as Error).toString()], 500, {});
   }
 };
+
+export function fetchWithCreds(input: RequestInfo, init: RequestInit) {
+  return fetcher(input, { credentials: 'include', ...init });
```

Comment:
```suggestion
  return fetcher(input, { ...init, credentials: 'include' });
```

Так будет корректнее, credentials всегда будет include

Reply:
+